pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/OperatorUtils.java

```diff
@@ -51,6 +52,7 @@ private OperatorUtils() {
    * @return Canonicalize form of the input function name
    */
   public static String canonicalizeFunctionName(String functionName) {
+    functionName = StringUtils.remove(functionName, " ");
```

Comment:
Ideally this should not even be allowed imo. Can you give an example ?

Reply:
calcite automatically converts multi-part functions like `IS NOT NULL` to that in SqlOp table.